src/main/java/com/internet/shop/controllers/ProductAddingController.java

```diff
@@ -0,0 +1,32 @@
+package com.internet.shop.controllers;
+
+import com.internet.shop.library.Injector;
+import com.internet.shop.model.Product;
+import com.internet.shop.service.interfaces.ProductService;
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+public class ProductAddingController extends HttpServlet {
```

Comment:
```suggestion
public class AddProductController extends HttpServlet {
```
let's change other controllers in the same style